src/Service/EscapeService.php

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Service;
+
+use Lmc\ApiFilter\Entity\Value;
+use Lmc\ApiFilter\Escape\EscapeInterface;
+
+class EscapeService implements EscapeInterface
```

Comment:
I'm (among few other things :D ) confused about this whole Escape stuff.

Why is it needed? For what context is the escaping being done? For SQL? If so, which SQL dialect? It is not obvious and I'm afraid this could lead to security vulnerabilities because of this confusion. Also implementing anything custom to do escaping should be the last option of all options, as this almost never ends well.

Reply:
Well this (among with few other things :D ) is mostly a _reminder_ that some espacping must be done. I think this whole espace problem will be solved when I add `prepared statements` which should be first thing to do after this PR is merged.

So I would not make anything about them now, because I hope they won't be there in the future.